src/evo/creditpool.cpp

```diff
@@ -239,14 +239,11 @@ bool CCreditPoolDiff::SetTarget(const CTransaction& tx, TxValidationState& state
 
     targetBalance = cbTx.creditPoolBalance;
 
-
     if (!llmq::utils::IsMNRewardReallocationActive(pindex)) return true;
 
-    CAmount blockReward = 0;
-    for (const CTxOut& txout : tx.vout) {
-        blockReward += txout.nValue;
-    }
-    platformReward = MasternodePayments::PlatformShare(GetMasternodePayment(cbTx.nHeight, blockReward, params.BRRHeight));
+    assert(blockReward.has_value());
```

Comment:
why are we asserting here? if it's optional we should be able to handle that?

Reply:
good point. how about 10fdfb030126985e534468b00f24e9232e22f521?